Sources/NIOHTTP2/HTTP2StreamChannel.swift

```diff
@@ -476,9 +476,15 @@ final class HTTP2StreamChannel: Channel, ChannelCore {
         fatalError("not implemented \(#function)")
     }
 
-    public func write0(_ data: NIOAny, promise: EventLoopPromise<Void>?) {
+    func wroteBytes(_ writeSize: Int) {
```

Comment:
We should assert that we're in the right event loop when we enter this function.